service/controller/cluster.go

```diff
@@ -271,18 +173,6 @@ func newClusterResources(config ClusterConfig) ([]resource.Interface, error) {
 		resources = append(resources, clusterSecretResource)
 	}
 
-	resources = append(resources,
-		// appResource manages the per cluster app-operator instance and the
-		// workload cluster apps.
-		appResource,
-		// appFinalizerResource removes finalizers after the per cluster
-		// app-operator instance has been deleted.
-		appFinalizerResource,
```

Comment:
Deletion logic changes so we don't delete app-operator until all app CRs are deleted.